portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/plus.vm

```diff
@@ -0,0 +1,360 @@
+
+        #parse("/vm/morpheus/snippets/styleable-snippet.vm")
+
+        #if ($pageTop)
+            <script>
+                var sakaiPortalWindow = "";
+                ${d}PBJQ(document).ready(function() {
+                    setupSkipNav();
+                });
+            </script>
+        #end ## END of IF ($pageTop)
+
+        <script>
+            // If we loaded a second jQuery, revert $ and jQuery to the first jQuery
+            if ( secondJQuery ) {
+                ${d}PBJQ = jQuery.noConflict(true); // Safely return $ to the tool jQuery
+                if ( ! dollarEmpty && !notJQuery ) jQuery = ${d};  // Return jQuery to point at the tool jQuery
+            }
+            if (notJQuery && !secondJQuery) {
+                ${d}PBJQ = jQuery.noConflict(true); // Safely return $ to the tool jQuery
+                //Just set jQuery to be the same thing since it didn't exist before
+                jQuery = ${d}PBJQ;
+
+            }
+            window.console && console.log('Portal script load complete PBJQ='+${d}PBJQ.fn.jquery+' jQuery='+jQuery.fn.jquery+' ${d}='+dollarVersion);
+
+            ${d}PBJQ(document).ready(function() {
+                setupSiteNav();
+                ## Moved IE aria label code to sakai.portal.ie.js // SAK-22308         
+            });
+        </script>
+
+        <script>
+        function portal_check_pnotify() {
+            ${d}PBJQ.getJSON( "/direct/portal/notify.json", function( data ) {
+                for(i=0; i<data.error.length; i++ ) {
+                     ${d}PBJQ(function(){
+                        PNotify.defaults.styling = 'bootstrap3';
+                        PNotify.defaults.icons = 'fontawesome4';
+                        PNotify.alert({
+                            title: '${rloader.getString("pnotify_notice")}',
+                            text: data.error[i],
+                            type: 'notice'
+                        });
+                    });
+                }
+            });
+        }
+    #if ($userWarning) 
+        ${d}PBJQ(document).ready(portal_check_pnotify);
+    #end
+        </script>
+
+        <!--[if lt IE 9]>
+        <script src="${pageSkinRepo}/${pageSkin}/js/ie/sakai.portal.ie.js$!{portalCDNQuery}"></script>
+<![endif]-->
+
+        <script src="${pageScriptPath}caps-lock-checker.js$!{portalCDNQuery}"></script>
+
+        <script src="$!{portalCDNPath}/portal/scripts/jumptotop.js$!{portalCDNQuery}"></script>
+    #if (${sakaiThemesEnabled})
+        <script src="$!{portalCDNPath}/portal/scripts/themeswitcher.js$!{portalCDNQuery}"></script>
+    #end
+        #parse("/vm/morpheus/includeAnalytics.vm")
+        #parse("/vm/morpheus/includePASystem.vm")
+
+        #parse("/vm/morpheus/includeCookieNotice.vm")
+
+        #parse("/vm/morpheus/includeGoogleTM-NoScript.vm")
+
+        ${includeExtraHead}
+
+        #if ($loggedIn)
+        <!-- Modal popup for profile picture changer -->
```

Comment:
should use the profile picture webcomponent

Reply:
This is a copy of the file site.vm in the same folder with as few changes as possible to support plus.   The complaint you have is also present in site.vm and I won't fix it in plus.vm without also fixing it identically in site.vm.  And frankly, when trinity comes in - both these files will get a lot of eyes and review.  I need to keep site.vm and plus.vm in parallel so that when trinity makes major changes to site.vm - I can get plus.vm working again post-trinity.